scripts/scr_weapon/scr_weapon.gml

```diff
@@ -1681,6 +1542,166 @@ global.weapons = {
 		"arp": 0,
 		"tags": ["vehicle", "turret", "volkite", "ancient"]
 	},
+// Hireling weapons
+	// Admech
+	"Hellgun": {
+		"abbreviation": "HllGun",
+		"attack": {
+			"standard": 30,
+			"master_crafted": 34,
+			"artifact": 38
+		},
+		"ammo": 10,
+		"range": 6.1,
+		"spli": 3,
+		"arp": 0,
+		"tags": [],
+		"ranged_hands": 1,
+		// "tags": ["arcane"], - how is it supposed to be arcane?
+		// ... (other attributes)
+	},
```

Comment:
Adjusted Hellgun according to various sources and relation to our current weapons.
```suggestion
	"Hellgun": {
		"abbreviation": "HllGun",
		"attack": {
			"standard": 30,
			"master_crafted": 35,
			"artifact": 40
		},
		"description": "Known as a Hellgun or Hot-shot Lasgun, these high-energy weapons are more potent and destructive than common Lasguns, drawing more power for a more penetrative shot, but also more complex, requiring reinforced barrels, thermal-cooling cells and gyro-stabilized power packs.",
		"ammo": 20,
		"range": 12,
		"spli": 3,
		"arp": 0,
		"tags": ["las"],
		"melee_hands": 0,
		"ranged_hands": 1,
	},
```